docker/sweepers_driver.py

```diff
@@ -80,21 +91,44 @@ def run_factory(sweeper_f: Callable) -> Callable:
         username=username,
         password=password,
         log_filepath='provenance.log',
-        log_level=logging.INFO  # TODO: pull this from LOGLEVEL env var
+        log_level=logging.INFO,  # TODO: pull this from LOGLEVEL env var
+        verify_host_certs = True if not dev_mode else False
     )
 
 
+def parse_cross_cluster_remotes(env_var_value: str | None) -> Iterable[Iterable[str]] | None:
+    """
+    Given the env var value specifying the CCS remote node-sets, return the value as a list of batches, where each batch
+    is a list of remotes to be processed at the same time.  Returns None if the value is not set, empty, or specifies an
+    empty list of remotes.
+    """
+
+    if not env_var_value:
+        return None
+
+    content = json.loads(env_var_value)
+    if len(content) < 1:
+        return None
+
+    return [batch.split() for batch in content]
+
+
 run_provenance = run_factory(provenance.run)
 run_ancestry = run_factory(ancestry.run)
 
-cross_cluster_remote_batches = os.environ.get("PROV_REMOTES") or None
-if cross_cluster_remote_batches is None:
+cross_cluster_remote_node_batches = parse_cross_cluster_remotes(os.environ.get("PROV_REMOTES"))
+log.info(f'Running sweepers')
+if cross_cluster_remote_node_batches is None:
+    log.info(f'No CCS remotes specified - running sweepers against base OpenSearch endpoint only')
     run_provenance()
     run_ancestry()
 else:
-    remote_nodesets = json.loads(cross_cluster_remote_batches)
-    for remote_nodeset in remote_nodesets:
-        cross_cluster_remotes = remote_nodeset.split()
-
+    f'CCS remotes specified: {json.dumps(cross_cluster_remote_node_batches)}'
```

Comment:
What's this string doing here? We missing a `log.info(…)`?

Reply:
Yanoit 😰